### ISSUE TITLE
Add instructions on updating `doap.rdf` in the how-to-release guide

### ISSUE THREAD

**Issue by Fokko** [apache/iceberg#9522]
### Feature Request / Improvement

Thanks for updating it in https://github.com/apache/iceberg/pull/9507, but this should be done as part of the release. I think we should add it to the [instructions](https://github.com/apache/iceberg/blob/main/site/docs/how-to-release.md).

### Query engine

None

**Comment by ajantha-bhat** [comment]
Agree. I was about to raise PR today for docs side about adding this info. But I can wait if anyone wants to contribute this.

**Comment by jbonofre** [comment]
I was about to do it, but maybe worth to wait the new doc/website.

**Comment by ajantha-bhat** [comment]
@jbonofre: Thanks. I have assigned the issue to you.
